### ISSUE TITLE
On the problem regarding the engine's inability to translate

### ISSUE THREAD

**Issue by KatsumiKougen** [lazyweb-ai/lazyweb-experiments#4]
To @jedwhite

I've read about your article on *`lazyweb.ai`* on r/Python. As far as I know, it is a search engine designed to resemble text messaging services.

To see if the engine does its job properly, I have **asked** it these "**test**" questions:

- [x] **What is "Kamen Rider"**<br>Result: [Wikipedia article](https://en.wikipedia.org/wiki/Kamen_Rider)
- [x] **Time in Hanoi**<br>Result: `12:41 PM on Monday, July 05, 2021 (GMT+7)`
- [ ] **Example in Python**<br>Result: [This link](https://www.w3schools.com/python/python_examples.asp)<br>Expected a code snippet, like this:<br>
```python
print("Hello World")
```
```python
from sys import stdout
stdout.write("Hello World\n")
```
- [x] **What is the difference between C, C++ and C#**<br>Result: [This link](https://csharp-station.com/understanding-the-differences-between-c-c-and-c/)
- [ ] **Result of "3 + 14"**<br>Result: `17 inches`<br>Expected `17`

In conclusion, the engine more or less works like you expect.

There is one fatal problem:<br>When a user want to translate something, i.e. enter *`Translate "四月は君の嘘"`* to make the engine translate this phrase from Japanese to English, it goes to a blank screen, as if unresponsive.

![Screenshot_20210705-123517.png](https://user-images.githubusercontent.com/77793743/124424965-1a3b7500-dd92-11eb-8280-6fdeaca4c463.png)

![Screenshot_20210705-123526.png](https://user-images.githubusercontent.com/77793743/124425008-26bfcd80-dd92-11eb-9ee6-6e50f5d9d662.png)

I am looking forward to your response on this phenomenon.

*@KatsumiKougen*

**Comment by jedwhite** [comment]
Hey thanks so much for stepping through that! Very appreciated. We can't see what people search and it's incredibly helpful to get these reports when things go wrong. We had a few reports about the translation intent throwing a client crash.

We've just deployed an update which includes the fix for that!

Please refresh the page (which will install the PWA update in the background), and then after your next browser restart (or all lazyweb tabs being closed) you should get the new version which should fix it up!

Thanks heaps :)

**Comment by jedwhite** [comment]
PS On the 3+14, that looks like a glitch too. I'm getting just 17 with or without spaces. Would you mind terribly please doing again, and then say "/bug" and click the little box to attach the payload please? That will let us see what the search payload was to help look into that too, as it was a little off as well :)
